Automattic/wp-calypso
# Multi Domains: Disable the experiment and enable multi-domains on onboarding

Related to https://github.com/Automattic/dotcom-forge/issues/4459

## Proposed Changes

* This PR disabled the multi-domains experiment on onboarding and enables multi-domains for all users.

## Testing Instructions



* Test that /start properly works with multi-domain selection
* Ensure /domains properly works with multi-domain selection

## Pre-merge Checklist
